Maestro/subscriptions.json

```diff
@@ -1852,30 +1852,6 @@
         }
       }
     },
-    // Automate opening PRs to merge dotnet org repos' release/2.2 changes into master branches
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/buildtools/blob/release/2.2/**/*",
```

Comment:
Let's leave this one alone, buildtools hasn't changed as much.

Reply:
Updated to only remove the source-build trigger.